test/products/getProductTest.js

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ *
+ *    Copyright 2018 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+'use strict';
+
+const assert = require('chai').assert;
+const setup = require('../lib/setupTest').setup;
+const sampleProductSearch = require('../resources/sample-product-search-single-product');
+const config = require('../lib/config').config;
+
+describe('magento getProduct', () => {
+
+    describe('Unit tests', () => {
+
+        // build the helper in the context of '.this' suite
+        setup(this, __dirname, 'getProduct');
+
+        it('returns an error for a missing expected parameter', () => {
+            return this.prepareReject(null).execute(null).then(result => {
+                assert.strictEqual(result.response.error.name, 'MissingPropertyError');
+            });
+        });
+
+        it('returns an error for a failed backend request', () => {
+            return this.prepareReject(undefined)
+                .execute(Object.assign({
+                    'id': 'someId"'
+                }, config))
+                .then(result => {
+                    assert.strictEqual(result.response.error.name, 'UnexpectedError');
+                });
+        });
+
+        it('performs a get product for an existing product', () => {
```

Comment:
`it('returns a product')`

Reply:
Again, the searchProducts test has some messages like `performs a profiled product search in debug mode`. If that is not a problem, why is this one?
If you care so strictly about this, how about we create a ticket to make sure all the current descriptions follow the same rule.

Reply:
In general, something being wrong at a different place, it's no excuse to make the same error again. That just adds stuff to the pile of technical depth.

However, this test checks if the action returns a product. The other test you mentioned, is supposed to do the same, but the focus is on the fact that the request is profiled with debug mode activated.